packages/mds-agency/request-handlers.ts

```diff
@@ -314,15 +338,15 @@ export const submitVehicleEvent = async (req: AgencyApiRequest, res: AgencyApiRe
   }
 }
 
-export const submitVehicleTelemetry = async (req: AgencyApiRequest, res: AgencyApiResponse) => {
+export const submitVehicleTelemetry = async (req: AgencyApiRequest, res: AgencySubmitVehicleTelemetryResponse) => {
   const start = Date.now()
 
   const { data } = req.body
   const { provider_id } = res.locals
   if (!provider_id) {
     res.status(400).send({
       error: 'bad_param',
-      error_description: 'bad or missing provider_id'
+      error_description: 'Bad or missing provider_id'
     })
     return
   }
```

Comment:
I think that this is just unnecessary, we should _always_ have a provider_id at this point